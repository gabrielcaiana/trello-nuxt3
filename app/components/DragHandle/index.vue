<template>
  <Icon
    class="drag-handle cursor-move text-white"
    name="ph:dots-six-vertical"
    weight="bold"
  />
</template>
