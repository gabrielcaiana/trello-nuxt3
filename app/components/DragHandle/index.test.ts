import { shallowMount } from '@vue/test-utils';
import { describe, it, expect } from 'vitest';
import DragHandle from './index.vue';

describe('Icon', () => {
  it('renders the correct icon', () => {
    const wrapper = shallowMount(DragHandle);
    expect(wrapper.classes()).toEqual(
      expect.arrayContaining(['drag-handle', 'cursor-move', 'text-white'])
    );
    expect(wrapper).toMatchSnapshot();
  });
});
