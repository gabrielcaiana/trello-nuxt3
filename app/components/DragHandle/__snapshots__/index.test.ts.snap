// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Icon > renders the correct icon 1`] = `
VueWrapper {
  "__app": {
    "_component": {
      "name": "VTU_ROOT",
      "render": [Function],
      "setup": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <icon
        class="drag-handle cursor-move text-white"
        name="ph:dots-six-vertical"
        weight="bold"
      />
    </div>,
    "_context": {
      "app": [Circular],
      "components": {
        "transition": {
          "name": "transition",
        },
        "transition-group": {
          "name": "transition-group",
        },
      },
      "config": {
        "compilerOptions": {},
        "errorHandler": undefined,
        "globalProperties": {},
        "isNativeTag": [Function],
        "optionMergeStrategies": {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": {},
      "emitsCache": WeakMap {},
      "mixins": [
        {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": {},
      "reload": [Function],
    },
    "_instance": {
      "a": null,
      "accessCache": {},
      "appContext": {
        "app": [Circular],
        "components": {
          "transition": {
            "name": "transition",
          },
          "transition-group": {
            "name": "transition-group",
          },
        },
        "config": {
          "compilerOptions": {},
          "errorHandler": undefined,
          "globalProperties": {},
          "isNativeTag": [Function],
          "optionMergeStrategies": {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": {},
        "emitsCache": WeakMap {},
        "mixins": [
          {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": {},
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": {
        "VTU_COMPONENT": {},
      },
      "da": null,
      "data": {},
      "devtoolsRawSetupState": {
        "VTU_COMPONENT": RefImpl {
          "__v_isRef": true,
          "__v_isShallow": false,
          "_rawValue": {},
          "_value": {},
          "dep": Dep {
            "activeLink": undefined,
            "computed": undefined,
            "key": undefined,
            "map": undefined,
            "sc": 0,
            "subs": undefined,
            "subsHead": undefined,
            "version": 1,
          },
        },
      },
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "cleanup": undefined,
        "deps": Link {
          "dep": Dep {
            "activeLink": undefined,
            "computed": undefined,
            "key": Symbol(Object iterate),
            "map": Map {
              Symbol(Object iterate) => [Circular],
              "ref" => Dep {
                "activeLink": undefined,
                "computed": undefined,
                "key": "ref",
                "map": [Circular],
                "sc": 1,
                "subs": Link {
                  "dep": [Circular],
                  "nextDep": Link {
                    "dep": Dep {
                      "activeLink": undefined,
                      "computed": undefined,
                      "key": Symbol(Object iterate),
                      "map": Map {
                        Symbol(Object iterate) => [Circular],
                      },
                      "sc": 1,
                      "subs": [Circular],
                      "subsHead": [Circular],
                      "version": 0,
                    },
                    "nextDep": undefined,
                    "nextSub": undefined,
                    "prevActiveLink": undefined,
                    "prevDep": [Circular],
                    "prevSub": undefined,
                    "sub": [Circular],
                    "version": 0,
                  },
                  "nextSub": undefined,
                  "prevActiveLink": undefined,
                  "prevDep": [Circular],
                  "prevSub": undefined,
                  "sub": [Circular],
                  "version": 0,
                },
                "subsHead": Link {
                  "dep": [Circular],
                  "nextDep": Link {
                    "dep": Dep {
                      "activeLink": undefined,
                      "computed": undefined,
                      "key": Symbol(Object iterate),
                      "map": Map {
                        Symbol(Object iterate) => [Circular],
                      },
                      "sc": 1,
                      "subs": [Circular],
                      "subsHead": [Circular],
                      "version": 0,
                    },
                    "nextDep": undefined,
                    "nextSub": undefined,
                    "prevActiveLink": undefined,
                    "prevDep": [Circular],
                    "prevSub": undefined,
                    "sub": [Circular],
                    "version": 0,
                  },
                  "nextSub": undefined,
                  "prevActiveLink": undefined,
                  "prevDep": [Circular],
                  "prevSub": undefined,
                  "sub": [Circular],
                  "version": 0,
                },
                "version": 0,
              },
            },
            "sc": 1,
            "subs": [Circular],
            "subsHead": [Circular],
            "version": 0,
          },
          "nextDep": Link {
            "dep": Dep {
              "activeLink": undefined,
              "computed": undefined,
              "key": "ref",
              "map": Map {
                Symbol(Object iterate) => Dep {
                  "activeLink": undefined,
                  "computed": undefined,
                  "key": Symbol(Object iterate),
                  "map": [Circular],
                  "sc": 1,
                  "subs": [Circular],
                  "subsHead": [Circular],
                  "version": 0,
                },
                "ref" => [Circular],
              },
              "sc": 1,
              "subs": [Circular],
              "subsHead": [Circular],
              "version": 0,
            },
            "nextDep": Link {
              "dep": Dep {
                "activeLink": undefined,
                "computed": undefined,
                "key": Symbol(Object iterate),
                "map": Map {
                  Symbol(Object iterate) => [Circular],
                },
                "sc": 1,
                "subs": [Circular],
                "subsHead": [Circular],
                "version": 0,
              },
              "nextDep": undefined,
              "nextSub": undefined,
              "prevActiveLink": undefined,
              "prevDep": [Circular],
              "prevSub": undefined,
              "sub": [Circular],
              "version": 0,
            },
            "nextSub": undefined,
            "prevActiveLink": undefined,
            "prevDep": [Circular],
            "prevSub": undefined,
            "sub": [Circular],
            "version": 0,
          },
          "nextSub": undefined,
          "prevActiveLink": undefined,
          "prevDep": undefined,
          "prevSub": undefined,
          "sub": [Circular],
          "version": 0,
        },
        "depsTail": Link {
          "dep": Dep {
            "activeLink": undefined,
            "computed": undefined,
            "key": Symbol(Object iterate),
            "map": Map {
              Symbol(Object iterate) => [Circular],
            },
            "sc": 1,
            "subs": [Circular],
            "subsHead": [Circular],
            "version": 0,
          },
          "nextDep": undefined,
          "nextSub": undefined,
          "prevActiveLink": undefined,
          "prevDep": Link {
            "dep": Dep {
              "activeLink": undefined,
              "computed": undefined,
              "key": "ref",
              "map": Map {
                Symbol(Object iterate) => Dep {
                  "activeLink": undefined,
                  "computed": undefined,
                  "key": Symbol(Object iterate),
                  "map": [Circular],
                  "sc": 1,
                  "subs": Link {
                    "dep": [Circular],
                    "nextDep": [Circular],
                    "nextSub": undefined,
                    "prevActiveLink": undefined,
                    "prevDep": undefined,
                    "prevSub": undefined,
                    "sub": [Circular],
                    "version": 0,
                  },
                  "subsHead": Link {
                    "dep": [Circular],
                    "nextDep": [Circular],
                    "nextSub": undefined,
                    "prevActiveLink": undefined,
                    "prevDep": undefined,
                    "prevSub": undefined,
                    "sub": [Circular],
                    "version": 0,
                  },
                  "version": 0,
                },
                "ref" => [Circular],
              },
              "sc": 1,
              "subs": [Circular],
              "subsHead": [Circular],
              "version": 0,
            },
            "nextDep": [Circular],
            "nextSub": undefined,
            "prevActiveLink": undefined,
            "prevDep": Link {
              "dep": Dep {
                "activeLink": undefined,
                "computed": undefined,
                "key": Symbol(Object iterate),
                "map": Map {
                  Symbol(Object iterate) => [Circular],
                  "ref" => Dep {
                    "activeLink": undefined,
                    "computed": undefined,
                    "key": "ref",
                    "map": [Circular],
                    "sc": 1,
                    "subs": [Circular],
                    "subsHead": [Circular],
                    "version": 0,
                  },
                },
                "sc": 1,
                "subs": [Circular],
                "subsHead": [Circular],
                "version": 0,
              },
              "nextDep": [Circular],
              "nextSub": undefined,
              "prevActiveLink": undefined,
              "prevDep": undefined,
              "prevSub": undefined,
              "sub": [Circular],
              "version": 0,
            },
            "prevSub": undefined,
            "sub": [Circular],
            "version": 0,
          },
          "prevSub": undefined,
          "sub": [Circular],
          "version": 0,
        },
        "flags": 37,
        "fn": [Function],
        "next": undefined,
        "onTrack": undefined,
        "onTrigger": undefined,
        "scheduler": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "ids": [
        "",
        0,
        0,
      ],
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "job": [Function],
      "m": null,
      "next": null,
      "parent": null,
      "props": {},
      "propsDefaults": {},
      "propsOptions": [
        {},
        [],
      ],
      "provides": {},
      "proxy": {
        "VTU_COMPONENT": {},
      },
      "refs": {
        "VTU_COMPONENT": {},
      },
      "render": [Function],
      "renderCache": [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "_active": true,
        "_isPaused": false,
        "cleanups": [],
        "detached": true,
        "effects": [
          ReactiveEffect {
            "cleanup": undefined,
            "deps": Link {
              "dep": Dep {
                "activeLink": undefined,
                "computed": undefined,
                "key": Symbol(Object iterate),
                "map": Map {
                  Symbol(Object iterate) => [Circular],
                  "ref" => Dep {
                    "activeLink": undefined,
                    "computed": undefined,
                    "key": "ref",
                    "map": [Circular],
                    "sc": 1,
                    "subs": Link {
                      "dep": [Circular],
                      "nextDep": Link {
                        "dep": Dep {
                          "activeLink": undefined,
                          "computed": undefined,
                          "key": Symbol(Object iterate),
                          "map": Map {
                            Symbol(Object iterate) => [Circular],
                          },
                          "sc": 1,
                          "subs": [Circular],
                          "subsHead": [Circular],
                          "version": 0,
                        },
                        "nextDep": undefined,
                        "nextSub": undefined,
                        "prevActiveLink": undefined,
                        "prevDep": [Circular],
                        "prevSub": undefined,
                        "sub": [Circular],
                        "version": 0,
                      },
                      "nextSub": undefined,
                      "prevActiveLink": undefined,
                      "prevDep": [Circular],
                      "prevSub": undefined,
                      "sub": [Circular],
                      "version": 0,
                    },
                    "subsHead": Link {
                      "dep": [Circular],
                      "nextDep": Link {
                        "dep": Dep {
                          "activeLink": undefined,
                          "computed": undefined,
                          "key": Symbol(Object iterate),
                          "map": Map {
                            Symbol(Object iterate) => [Circular],
                          },
                          "sc": 1,
                          "subs": [Circular],
                          "subsHead": [Circular],
                          "version": 0,
                        },
                        "nextDep": undefined,
                        "nextSub": undefined,
                        "prevActiveLink": undefined,
                        "prevDep": [Circular],
                        "prevSub": undefined,
                        "sub": [Circular],
                        "version": 0,
                      },
                      "nextSub": undefined,
                      "prevActiveLink": undefined,
                      "prevDep": [Circular],
                      "prevSub": undefined,
                      "sub": [Circular],
                      "version": 0,
                    },
                    "version": 0,
                  },
                },
                "sc": 1,
                "subs": [Circular],
                "subsHead": [Circular],
                "version": 0,
              },
              "nextDep": Link {
                "dep": Dep {
                  "activeLink": undefined,
                  "computed": undefined,
                  "key": "ref",
                  "map": Map {
                    Symbol(Object iterate) => Dep {
                      "activeLink": undefined,
                      "computed": undefined,
                      "key": Symbol(Object iterate),
                      "map": [Circular],
                      "sc": 1,
                      "subs": [Circular],
                      "subsHead": [Circular],
                      "version": 0,
                    },
                    "ref" => [Circular],
                  },
                  "sc": 1,
                  "subs": [Circular],
                  "subsHead": [Circular],
                  "version": 0,
                },
                "nextDep": Link {
                  "dep": Dep {
                    "activeLink": undefined,
                    "computed": undefined,
                    "key": Symbol(Object iterate),
                    "map": Map {
                      Symbol(Object iterate) => [Circular],
                    },
                    "sc": 1,
                    "subs": [Circular],
                    "subsHead": [Circular],
                    "version": 0,
                  },
                  "nextDep": undefined,
                  "nextSub": undefined,
                  "prevActiveLink": undefined,
                  "prevDep": [Circular],
                  "prevSub": undefined,
                  "sub": [Circular],
                  "version": 0,
                },
                "nextSub": undefined,
                "prevActiveLink": undefined,
                "prevDep": [Circular],
                "prevSub": undefined,
                "sub": [Circular],
                "version": 0,
              },
              "nextSub": undefined,
              "prevActiveLink": undefined,
              "prevDep": undefined,
              "prevSub": undefined,
              "sub": [Circular],
              "version": 0,
            },
            "depsTail": Link {
              "dep": Dep {
                "activeLink": undefined,
                "computed": undefined,
                "key": Symbol(Object iterate),
                "map": Map {
                  Symbol(Object iterate) => [Circular],
                },
                "sc": 1,
                "subs": [Circular],
                "subsHead": [Circular],
                "version": 0,
              },
              "nextDep": undefined,
              "nextSub": undefined,
              "prevActiveLink": undefined,
              "prevDep": Link {
                "dep": Dep {
                  "activeLink": undefined,
                  "computed": undefined,
                  "key": "ref",
                  "map": Map {
                    Symbol(Object iterate) => Dep {
                      "activeLink": undefined,
                      "computed": undefined,
                      "key": Symbol(Object iterate),
                      "map": [Circular],
                      "sc": 1,
                      "subs": Link {
                        "dep": [Circular],
                        "nextDep": [Circular],
                        "nextSub": undefined,
                        "prevActiveLink": undefined,
                        "prevDep": undefined,
                        "prevSub": undefined,
                        "sub": [Circular],
                        "version": 0,
                      },
                      "subsHead": Link {
                        "dep": [Circular],
                        "nextDep": [Circular],
                        "nextSub": undefined,
                        "prevActiveLink": undefined,
                        "prevDep": undefined,
                        "prevSub": undefined,
                        "sub": [Circular],
                        "version": 0,
                      },
                      "version": 0,
                    },
                    "ref" => [Circular],
                  },
                  "sc": 1,
                  "subs": [Circular],
                  "subsHead": [Circular],
                  "version": 0,
                },
                "nextDep": [Circular],
                "nextSub": undefined,
                "prevActiveLink": undefined,
                "prevDep": Link {
                  "dep": Dep {
                    "activeLink": undefined,
                    "computed": undefined,
                    "key": Symbol(Object iterate),
                    "map": Map {
                      Symbol(Object iterate) => [Circular],
                      "ref" => Dep {
                        "activeLink": undefined,
                        "computed": undefined,
                        "key": "ref",
                        "map": [Circular],
                        "sc": 1,
                        "subs": [Circular],
                        "subsHead": [Circular],
                        "version": 0,
                      },
                    },
                    "sc": 1,
                    "subs": [Circular],
                    "subsHead": [Circular],
                    "version": 0,
                  },
                  "nextDep": [Circular],
                  "nextSub": undefined,
                  "prevActiveLink": undefined,
                  "prevDep": undefined,
                  "prevSub": undefined,
                  "sub": [Circular],
                  "version": 0,
                },
                "prevSub": undefined,
                "sub": [Circular],
                "version": 0,
              },
              "prevSub": undefined,
              "sub": [Circular],
              "version": 0,
            },
            "flags": 37,
            "fn": [Function],
            "next": undefined,
            "onTrack": undefined,
            "onTrigger": undefined,
            "scheduler": [Function],
          },
        ],
        "parent": undefined,
      },
      "setupContext": null,
      "setupState": {
        "VTU_COMPONENT": {},
      },
      "slots": {},
      "sp": null,
      "subTree": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": {
          "a": null,
          "accessCache": {
            "@@__IMMUTABLE_ITERABLE__@@": 0,
            "@@__IMMUTABLE_RECORD__@@": 0,
            "__v_isReadonly": 0,
            "__v_isRef": 0,
            "__v_isShallow": 0,
            "__v_raw": 0,
            "_isMockFunction": 0,
            "constructor": 0,
            "toJSON": 0,
            Symbol(Symbol.toStringTag): 0,
          },
          "appContext": {
            "app": [Circular],
            "components": {
              "transition": {
                "name": "transition",
              },
              "transition-group": {
                "name": "transition-group",
              },
            },
            "config": {
              "compilerOptions": {},
              "errorHandler": undefined,
              "globalProperties": {},
              "isNativeTag": [Function],
              "optionMergeStrategies": {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": {},
            "emitsCache": WeakMap {},
            "mixins": [
              {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": {},
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": {},
          "ctx": {},
          "da": null,
          "data": {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "cleanup": undefined,
            "deps": undefined,
            "depsTail": undefined,
            "flags": 37,
            "fn": [Function],
            "next": undefined,
            "onTrack": undefined,
            "onTrigger": undefined,
            "scheduler": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": null,
          "exposed": null,
          "ids": [
            "",
            0,
            0,
          ],
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "job": [Function],
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": {},
          "propsDefaults": {},
          "propsOptions": [
            {},
            [],
          ],
          "provides": {},
          "proxy": {},
          "refs": {},
          "render": [Function],
          "renderCache": [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "_active": true,
            "_isPaused": false,
            "cleanups": [],
            "detached": true,
            "effects": [
              ReactiveEffect {
                "cleanup": undefined,
                "deps": undefined,
                "depsTail": undefined,
                "flags": 37,
                "fn": [Function],
                "next": undefined,
                "onTrack": undefined,
                "onTrigger": undefined,
                "scheduler": [Function],
              },
            ],
            "parent": undefined,
          },
          "setupContext": null,
          "setupState": {},
          "slots": {},
          "sp": null,
          "subTree": {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": null,
            "appContext": null,
            "children": null,
            "component": null,
            "ctx": [Circular],
            "dirs": null,
            "dynamicChildren": [],
            "dynamicProps": null,
            "el": <icon
              class="drag-handle cursor-move text-white"
              name="ph:dots-six-vertical"
              weight="bold"
            />,
            "key": null,
            "patchFlag": 0,
            "props": {
              "class": "drag-handle cursor-move text-white",
              "name": "ph:dots-six-vertical",
              "weight": "bold",
            },
            "ref": null,
            "scopeId": null,
            "shapeFlag": 1,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "targetStart": null,
            "transition": null,
            "type": "Icon",
          },
          "suspense": null,
          "suspenseId": 0,
          "type": {
            "__file": "C:/Users/Pichau/www/personal/trello-nuxt3/components/DragHandle/index.vue",
            "components": {},
            "render": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "ctx": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <icon
          class="drag-handle cursor-move text-white"
          name="ph:dots-six-vertical"
          weight="bold"
        />,
        "key": null,
        "patchFlag": 0,
        "props": {
          "ref": "VTU_COMPONENT",
        },
        "ref": {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "targetStart": null,
        "transition": null,
        "type": {
          "__file": "C:/Users/Pichau/www/personal/trello-nuxt3/components/DragHandle/index.vue",
          "components": {},
          "render": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": {
        "name": "VTU_ROOT",
        "render": [Function],
        "setup": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": {
          "app": [Circular],
          "components": {
            "transition": {
              "name": "transition",
            },
            "transition-group": {
              "name": "transition-group",
            },
          },
          "config": {
            "compilerOptions": {},
            "errorHandler": undefined,
            "globalProperties": {},
            "isNativeTag": [Function],
            "optionMergeStrategies": {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": {},
          "emitsCache": WeakMap {},
          "mixins": [
            {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "ctx": null,
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <icon
          class="drag-handle cursor-move text-white"
          name="ph:dots-six-vertical"
          weight="bold"
        />,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "targetStart": null,
        "transition": null,
        "type": {
          "name": "VTU_ROOT",
          "render": [Function],
          "setup": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": {
      "compilerOptions": {},
      "errorHandler": undefined,
      "globalProperties": {},
      "isNativeTag": [Function],
      "optionMergeStrategies": {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "onUnmount": [Function],
    "provide": [Function],
    "runWithContext": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.5.13",
  },
  "__setProps": [Function],
  "cleanUpCallbacks": [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
  "componentVM": {},
  "isDisabled": [Function],
  "rootVM": {
    "VTU_COMPONENT": {},
  },
  "wrapperElement": <icon
    class="drag-handle cursor-move text-white"
    name="ph:dots-six-vertical"
    weight="bold"
  />,
}
`;
